packages/slate-editor/jest.config.cjs

```diff
@@ -1,13 +1,21 @@
+const path = require('path');
+
 module.exports = {
     coveragePathIgnorePatterns: ['/node_modules/', '/build/'],
     extensionsToTreatAsEsm: ['.ts', '.tsx'],
     globals: {
         'ts-jest': {
             useESM: true,
+            isolatedModules: true,
             tsconfig: `${__dirname}/tsconfig.jest.json`,
         },
     },
     moduleNameMapper: {
+        '@prezly/slate-tables': path.join(__dirname, '../slate-tables/src'),
```

Comment:
There is a problem with `jsdom` - it doesn't resolve modules like nodejs. Do not know why